src/components/entity/state-badge.ts

```diff
@@ -111,12 +111,21 @@ class StateBadge extends LitElement {
         transition: color 0.3s ease-in-out, filter 0.3s ease-in-out;
       }
 
-      /* Color the icon if light or sun is on */
-      ha-icon[data-domain="light"][data-state="on"],
-      ha-icon[data-domain="switch"][data-state="on"],
       ha-icon[data-domain="binary_sensor"][data-state="on"],
+      ha-icon[data-domain="cover"][data-state="open"],
+      ha-icon[data-domain="climate"][data-state="cooling"],
+      ha-icon[data-domain="climate"][data-state="heating"],
+      ha-icon[data-domain="device_tracker"][data-state="home"],
       ha-icon[data-domain="fan"][data-state="on"],
-      ha-icon[data-domain="sun"][data-state="above_horizon"] {
+      ha-icon[data-domain="light"][data-state="on"],
+      ha-icon[data-domain="input_boolean"][data-state="on"],
+      ha-icon[data-domain="lock"][data-state="unlocked"],
+      ha-icon[data-domain="media_player"][data-state="paused"],
```

Comment:
This is not an active state?

Reply:
I disagree. Something is actively paused. The device is still on and attention should be drawn to it over idle/off